spec/services/schedule_builder_spec.rb

```diff
@@ -1,10 +1,20 @@
 describe ScheduleBuilder do
+  let(:start_time) { Time.current }
+  let(:end_time) { start_time + 1.hour }
   let(:rules) { [{ type: 'weekly', day: 'monday' }] }
 
-  subject { ScheduleBuilder.call(rules: rules) }
+  subject {
+    ScheduleBuilder.call(
+      start_time: start_time,
+      end_time: end_time,
+      rules: rules
+    )
+  }
```

Comment:
Ah-ha, a give away that rubocop -a hasn't been run :stuck_out_tongue_winking_eye: